cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/model/ClusterModel.java

```diff
@@ -1391,4 +1409,16 @@ private void refreshCapacity() {
       _clusterCapacity[r.id()] = capacity;
     }
   }
+
+  private boolean sanityCheckPartitionInfoConsistency(PartitionInfo partitionInfo) {
```

Comment:
The name of this function is not very clear -- i.e. looks like it checks whether the cluster model has the same partition having the same replicas with the passed partition info.

Can we rename this function to make that more clear? Maybe something like `hasSameReplicasFor(...)`?